src/Classes/PassiveSpecListControl.lua

```diff
@@ -19,25 +21,49 @@ local PassiveSpecListClass = newClass("PassiveSpecListControl", "ListControl", f
 		self:RenameSpec(newSpec, "Copy Tree", true)
 	end)
 	self.controls.copy.enabled = function()
-		return self.selValue ~= nil
+		return self.selValue ~= nil and #self.selections == 1
 	end
 	self.controls.delete = new("ButtonControl", {"LEFT",self.controls.copy,"RIGHT"}, 4, 0, 60, 18, "Delete", function()
-		self:OnSelDelete(self.selIndex, self.selValue)
+		self:OnSelDelete()
```

Comment:
Why not just pass self.selections, here?

Reply:
Cause it's an internal variable that doesn't need passing